apps/atomy-q/API/app/Console/Commands/VerifyStorageDiskCommand.php

```
@@ -0,0 +1,96 @@
+    protected $signature = 'atomy:verify-storage-disk
+                            {--disk= : Filesystem disk to verify (defaults to filesystems.default)}
+                            {--path-prefix=alpha-storage-smoke : Path prefix for the smoke object}';
+
+    protected $description = 'Verify the configured storage disk can write and read a small object';
+
+    public function handle(): int
+    {
+        $disk = $this->resolveDisk();
+        $pathPrefix = $this->resolvePathPrefix();
+        $timestamp = now();
+        $path = $this->buildPath($pathPrefix, $timestamp, (string) Str::ulid());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Str' in method 'handle'. (undefined)

(StaticAccess)